stubs/pyxdg/xdg/IconTheme.pyi

```diff
@@ -0,0 +1,59 @@
+from _typeshed import Unused
+from collections.abc import Collection
+from os import PathLike
+from typing import AnyStr
+
+from xdg.IniFile import IniFile
+
+basedir: Unused  # This is something that shouldn't be included since it's
```

Comment:
Incomplete sentence. And if it shouldn't be included, maybe we should just omit it from the stub?

Reply:
I'm new to making stubs — when I omitted it, stubtest started complaining. I can only see a way to disable this for the whole project at a command-line level, not for just a single item. What's the best way to handle this?

(The rest of the sentence is that it's not part of the public API but just a result of a for loop in the file)

Reply:
You can add an allowlist file that lists specific names; you'll see examples of that in several other third-party stubs.

Reply:
E.g. https://github.com/python/typeshed/blob/main/stubs/SQLAlchemy/%40tests/stubtest_allowlist.txt as an example :)

Reply:
Ahh never mind, got it. Just needed to add it to the allow list.